charts/pgcat/Chart.yaml

```diff
@@ -5,4 +5,4 @@ maintainers:
   - name: Wildcard
     email: support@w6d.io
 appVersion: "1.2.0"
-version: 0.2.1
+version: 0.3.0
```

Comment:
Not sure if semver is strict here, but bumped to 0.3.0 since new, non-breaking functionality is added. Happy to make this 0.2.2 if that's preferred.